src/System.Device.Gpio/System/Device/Gpio/SafeLineHandle.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace System.Device.Gpio
+{
+    class SafeLineHandle : SafeHandle
+    {
+        internal SafeLineHandle() : base(IntPtr.Zero, false) { }
+
+        override protected bool ReleaseHandle()
+        {
+            return true;
```

Comment:
Don't we need to call Interop.gpiod_line_release here?

Reply:
Interop.gpiod_line_release() receives this SafeLineHandle type, basically that is for releasing pin programmatically from within API, for releasing safe handle here there should be method it should use IntpPtr. When the chip is closed all resources will be closed including all lines used, so we don't need to call release here

Reply:
do we even need this handle then? it doesn't seem to be doing anything useful

Reply:
We need it because some other methods require line pointer (`struct gpiod_line *line`) as argument ike : `int gpiod_line_set_value(struct gpiod_line *line, int value)`